src/CarrierLogo/README.md

```diff
@@ -13,6 +13,7 @@ Table below contains all types of the props available in CarrierLogo component.
 | Name          | Type                             | Default         | Description                      |
 | :------------ | :------------------------------- | :-------------- | :------------------------------- |
 | **carriers**  | [`Carrier[]`](#carrier)          |                 | The content of the CarrierLogo, passed as array of objects.
+| customSource  | [`Object`](#custom-source)        |                 | Optional object property for using different source of images.
```

Comment:
Should be enough to pass just URL ? this is overthinking, we should build URLs for them, it's `custom`